[2023-05-28 15:19:10,133] logger_util.py->write_log line:61 [INFO] -------------接口请求开始-----------
[2023-05-28 15:19:10,133] logger_util.py->write_log line:61 [INFO] 接口名称：创建归档表
[2023-05-28 15:19:10,133] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-05-28 15:19:10,133] logger_util.py->write_log line:61 [INFO] 接口路径：http://192.168.4.12:18000/api/v1/dataconfig/filingtable
[2023-05-28 15:19:10,134] logger_util.py->write_log line:61 [INFO] 请求头：{'Content-Type': 'application/json;charset=UTF-8', 'X-Project': 'b32d81056e124620b809e92f49cdf3d4'}
[2023-05-28 15:19:10,134] logger_util.py->write_log line:61 [INFO] 请求参数：{'folderId': '00000000-0000-0000-0000-000000000000', 'name': '归档表39', 'sort': 2}
[2023-05-28 15:19:10,134] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-05-28 15:19:10,279] logger_util.py->write_log line:61 [INFO] 预期结果：[{'equals': {'status_code': 200}}, {'equals': {'sort': 2}}, {'contains': 'id'}]
[2023-05-28 15:19:10,279] logger_util.py->write_log line:61 [INFO] 实际结果：{'id': '879f3504-4801-49c1-8df4-f549272afec7', 'folderId': '00000000-0000-0000-0000-000000000000', 'name': '归档表39', 'sort': 2, 'createDateTime': '2023-05-28T15:19:10.2812414+08:00', 'operationTypes': [1, 2, 3, 4], 'functionTypes': [1, 2, 3, 4], 'extendedHeaders': ['记录时间'], 'cacheConfig_EnableCache': False, 'cacheConfig_QuantityLimit': 0, 'cacheConfig_TimeSpanLimit': 0, 'cacheConfig_TimeSpanLimitUnit': 0, 'cacheConfig_ClearCache': '', 'storageConfig_ScheduledClear': False, 'storageConfig_ClearTriggerTime': None, 'storageConfig_ClearFrequency': 0, 'storageConfig_ClearFrequencyUnit': 0, 'storageConfig_ClearRetain': 0, 'storageConfig_ClearRetainUnit': 0, 'storageConfig_ScheduledBackup': False, 'storageConfig_BackupTriggerTime': None, 'storageConfig_BackupFrequency': 0, 'storageConfig_BackupFrequencyUnit': 0, 'storageConfig_BackupPath': ''}
[2023-05-28 15:19:10,279] logger_util.py->write_log line:61 [INFO] 接口请求成功！
[2023-05-28 15:19:10,279] logger_util.py->write_log line:61 [INFO] -------------接口请求结束-----------

[2023-05-28 15:19:10,289] logger_util.py->write_log line:61 [INFO] -------------接口请求开始-----------
[2023-05-28 15:19:10,289] logger_util.py->write_log line:61 [INFO] 接口名称：创建聚合表
[2023-05-28 15:19:10,289] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-05-28 15:19:10,289] logger_util.py->write_log line:61 [INFO] 接口路径：http://192.168.4.12:18000/api/v1/dataconfig/aggregatetable
[2023-05-28 15:19:10,290] logger_util.py->write_log line:61 [INFO] 请求头：{'Content-Type': 'application/json;charset=UTF-8', 'X-Project': 'b32d81056e124620b809e92f49cdf3d4'}
[2023-05-28 15:19:10,290] logger_util.py->write_log line:61 [INFO] 请求参数：{'folderId': '00000000-0000-0000-0000-000000000000', 'name': '聚合表7', 'sort': 3, 'tableId': '879f3504-4801-49c1-8df4-f549272afec7', 'nodeType': 3}
[2023-05-28 15:19:10,290] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-05-28 15:19:10,392] logger_util.py->write_log line:61 [INFO] 预期结果：[{'equals': {'status_code': 200}}, {'equals': {'sort': 3}}, {'contains': 'id'}]
[2023-05-28 15:19:10,392] logger_util.py->write_log line:61 [INFO] 实际结果：{'id': '14814674-308c-4a29-8bc5-75a9da014f1e', 'folderId': '00000000-0000-0000-0000-000000000000', 'relativeTableId': '879f3504-4801-49c1-8df4-f549272afec7', 'relativeType': 3, 'name': '聚合表7', 'sort': 3, 'createDateTime': '2023-05-28T15:19:10.406299+08:00', 'preConditionType': 0, 'operationTypes': [1, 2, 3, 4], 'functionTypes': [2, 3, 4], 'extendedHeaders': ['记录时间'], 'storageConfig_ScheduledClear': False, 'storageConfig_ClearTriggerTime': None, 'storageConfig_ClearFrequency': 0, 'storageConfig_ClearFrequencyUnit': 0, 'storageConfig_ClearRetain': 0, 'storageConfig_ClearRetainUnit': 0, 'storageConfig_ScheduledBackup': False, 'storageConfig_BackupTriggerTime': None, 'storageConfig_BackupFrequency': 0, 'storageConfig_BackupFrequencyUnit': 0, 'storageConfig_BackupPath': ''}
[2023-05-28 15:19:10,392] logger_util.py->write_log line:61 [INFO] 接口请求成功！
[2023-05-28 15:19:10,392] logger_util.py->write_log line:61 [INFO] -------------接口请求结束-----------

[2023-05-28 15:19:10,402] logger_util.py->write_log line:61 [INFO] -------------接口请求开始-----------
[2023-05-28 15:19:10,402] logger_util.py->write_log line:61 [INFO] 接口名称：获取聚合表
[2023-05-28 15:19:10,402] logger_util.py->write_log line:61 [INFO] 接口方式：get
[2023-05-28 15:19:10,402] logger_util.py->write_log line:61 [INFO] 接口路径：http://192.168.4.12:18000/api/v1/dataconfig/aggregatetable/14814674-308c-4a29-8bc5-75a9da014f1e
[2023-05-28 15:19:10,402] logger_util.py->write_log line:61 [INFO] 请求头：{'Content-Type': 'application/json;charset=UTF-8', 'X-Project': 'b32d81056e124620b809e92f49cdf3d4'}
[2023-05-28 15:19:10,402] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-05-28 15:19:10,413] logger_util.py->write_log line:61 [INFO] 预期结果：[{'equals': {'status_code': 200}}, {'contains': 'id'}]
[2023-05-28 15:19:10,413] logger_util.py->write_log line:61 [INFO] 实际结果：{'id': '14814674-308c-4a29-8bc5-75a9da014f1e', 'folderId': '00000000-0000-0000-0000-000000000000', 'relativeTableId': '879f3504-4801-49c1-8df4-f549272afec7', 'relativeType': 3, 'name': '聚合表7', 'createDateTime': '2023-05-28T15:19:10.406299', 'preConditionType': 0, 'relativeTableName': None}
[2023-05-28 15:19:10,413] logger_util.py->write_log line:61 [INFO] 接口请求成功！
[2023-05-28 15:19:10,413] logger_util.py->write_log line:61 [INFO] -------------接口请求结束-----------

[2023-05-28 15:19:10,419] logger_util.py->write_log line:61 [INFO] -------------接口请求开始-----------
[2023-05-28 15:19:10,419] logger_util.py->write_log line:61 [INFO] 接口名称：获取所有聚合表
[2023-05-28 15:19:10,419] logger_util.py->write_log line:61 [INFO] 接口方式：get
[2023-05-28 15:19:10,419] logger_util.py->write_log line:61 [INFO] 接口路径：http://192.168.4.12:18000/api/v1/dataconfig/aggregatetable
[2023-05-28 15:19:10,419] logger_util.py->write_log line:61 [INFO] 请求头：{'Content-Type': 'application/json;charset=UTF-8', 'X-Project': 'b32d81056e124620b809e92f49cdf3d4'}
[2023-05-28 15:19:10,419] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-05-28 15:19:10,433] logger_util.py->error_log line:65 [INFO] 分析YAML文件异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\2TestCMS2.0\common\requests_util.py", line 220, in analysis_yaml
    value_data = int(value_data[1])
IndexError: list index out of range

